{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "total_count": 1,
    "last_updated": "2025-10-02T21:49:55.417485",
    "severity_distribution": {
      "critical": 0,
      "high": 0,
      "medium": 0,
      "low": 0,
      "none": 1
    }
  },
  "cves": [
    {
      "id": "CVE-2025-61668",
      "publishedDate": "2025-10-02T21:46:32.975Z",
      "lastModifiedDate": "2025-10-02T21:46:32.975Z",
      "description": "Volto is a ReactJS-based frontend for the Plone Content Management System. Versions 16.34.0 and below, 17.0.0 through 17.22.1, 18.0.0 through 18.27.1, and 19.0.0-alpha.1 through 19.0.0-alpha.5, an anonymous user could cause the NodeJS server part of Volto to quit with an error when visiting a specific URL. This issue is fixed in versions 16.34.1, 17.22.2, 18.27.2 and 19.0.0-alpha.6.",
      "severity": "N/A",
      "references": [
        {
          "url": "https://github.com/plone/volto/security/advisories/GHSA-m8rj-ppph-mj33",
          "type": "reference"
        },
        {
          "url": "https://github.com/plone/volto/pull/7412",
          "type": "reference"
        },
        {
          "url": "https://github.com/plone/volto/pull/7413",
          "type": "reference"
        },
        {
          "url": "https://github.com/plone/volto/commit/58d9f82d2d50ca9a87edbe16fed91762e57c109c",
          "type": "reference"
        },
        {
          "url": "https://github.com/plone/volto/releases/tag/16.34.1",
          "type": "reference"
        },
        {
          "url": "https://github.com/plone/volto/releases/tag/17.22.2",
          "type": "reference"
        },
        {
          "url": "https://github.com/plone/volto/releases/tag/19.0.0-alpha.6",
          "type": "reference"
        },
        {
          "url": "http://github.com/plone/volto/releases/tag/18.27.2",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "vendor": "plone",
          "product": "volto",
          "versions": [
            {
              "version": "< 16.34.1",
              "status": "affected"
            },
            {
              "version": ">= 17.0.0, < 17.22.2",
              "status": "affected"
            },
            {
              "version": ">= 18.0.0, < 18.27.2",
              "status": "affected"
            },
            {
              "version": ">= 19.0.0-alpha.1, < 19.0.0-alpha.6",
              "status": "affected"
            }
          ]
        }
      ],
      "problemType": [
        "CWE-476: NULL Pointer Dereference",
        "CWE-754: Improper Check for Unusual or Exceptional Conditions"
      ],
      "fix_suggestion": "无法生成修复建议"
    }
  ]
}